CSharp_Course_Project/Task1_JadenSmithCase.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+
+namespace CSharp_Course_Project
+{
+    class Task1_JadenSmithCase
+    {
+        public static void JadenSmithCase()
+        {
+            var jadenCasedText = FirstLetterToUpperCase();
```

Comment:
When you invoke a static method, please use name of class.

Reply:
I don't quite understand this comment